### PR TITLE
Release 09 09 2021

### PR DESCRIPTION
**Pull request by apaparazzi0329**

### CODE REVIEW

File: Sources/DiscoveryV2/Models/ProjectDetails.swift [hunk 1]
```diff
@@ -22,12 +22,17 @@ import Foundation
...
+     The type of project.
+     The `content_intelligence` type is a *Document Retrieval for Contracts* project and the `other` type is a *Custom*
+     project.
+     The `content_mining` and `content_intelligence` types are available with Premium plan managed deployments and
+     installed deployments only.
      */
     public enum TypeEnum: String {
         case documentRetrieval = "document_retrieval"
-        case answerRetrieval = "answer_retrieval"
+        case conversationalSearch = "conversational_search"
```
Comment (repjarms) [inline Sources/DiscoveryV2/Models/ProjectDetails.swift:33]: Just confirming that we are OK with changing `answer_retrieval` to `conversational_search` here.

File: Sources/DiscoveryV2/Models/ProjectListDetails.swift [hunk 2]
```diff
@@ -22,12 +22,17 @@ import Foundation
...
+     The type of project.
+     The `content_intelligence` type is a *Document Retrieval for Contracts* project and the `other` type is a *Custom*
+     project.
+     The `content_mining` and `content_intelligence` types are available with Premium plan managed deployments and
+     installed deployments only.
      */
     public enum TypeEnum: String {
         case documentRetrieval = "document_retrieval"
-        case answerRetrieval = "answer_retrieval"
+        case conversationalSearch = "conversational_search"
```
Comment (repjarms) [inline Sources/DiscoveryV2/Models/ProjectListDetails.swift:33]: Same as above

File: Tests/NaturalLanguageUnderstandingV1Tests/NaturalLanguageUnderstandingV1Tests.swift [hunk 3]
```diff
@@ -566,14 +566,14 @@ class NaturalLanguageUnderstandingTests: XCTestCase {
...
-            categories: CategoriesOptions(),
             concepts: ConceptsOptions(limit: 5),
             emotion: EmotionOptions(document: true, targets: ["happy"]),
             entities: EntitiesOptions(limit: 5, mentions: true, model: "en-news", sentiment: true, emotion: true),
             keywords: KeywordsOptions(limit: 5, sentiment: true, emotion: true),
             relations: RelationsOptions(model: "en-news"),
             semanticRoles: SemanticRolesOptions(limit: 5, keywords: true, entities: true),
-            sentiment: SentimentOptions(document: true, targets: ["happy"])
+            sentiment: SentimentOptions(document: true, targets: ["happy"]),
+            categories: CategoriesOptions()
```
Comment (repjarms) [inline Tests/NaturalLanguageUnderstandingV1Tests/NaturalLanguageUnderstandingV1Tests.swift:576]: this seems suspicious - why is the parameter order changing?

**Review by repjarms — COMMENTED** [review]
@apaparazzi0329 I left a couple questions on the review before I approve.

File: Sources/DiscoveryV2/Models/ProjectDetails.swift (hunk 1 above)
Comment (apaparazzi0329) [reply Sources/DiscoveryV2/Models/ProjectDetails.swift:33]: Yes, answer_retrieval is no longer a valid option

File: Sources/DiscoveryV2/Models/ProjectListDetails.swift (hunk 2 above)
Comment (apaparazzi0329) [reply Sources/DiscoveryV2/Models/ProjectListDetails.swift:33]: Yes, answer_retrieval is no longer a valid option

File: Tests/NaturalLanguageUnderstandingV1Tests/NaturalLanguageUnderstandingV1Tests.swift (hunk 3 above)
Comment (apaparazzi0329) [reply Tests/NaturalLanguageUnderstandingV1Tests/NaturalLanguageUnderstandingV1Tests.swift:576]: This was actually a mistake that was not addressed in the last release. It was only an issue with the test though

**Review by repjarms — APPROVED** [review]

**Pull request merged**